client/commonFramework/show-completion.js

```diff
@@ -3,9 +3,21 @@ window.common = (function(global) {
     $,
     moment,
     ga = (() => {}),
+    localStorage,
     common = { init: [] }
   } = global;
 
+  function linkGithubHandler() {
+    if (
+      !localStorage ||
+      typeof localStorage.setItem !== 'function'
+    ) {
+      console.log('unable to save to storage');
```

Comment:
I think you're missing a `return` statement here, otherwise if `localStorage` object won't be available (or `localStorage.setItem` won't be a function), your code will try to set an item anyway (line 17).